Header header

float64[3] y
float64[3] yhat
float64[3] error
float64[3] XYZ
float64[3] XYZhat
float64[3] XYZerror
float64[4] q
float64[4] qhat
float64[4] qError
float64[3] vc
float64[3] wc
float64[3] vt
float64[3] wt
float64[7] phi
float64[7] phiHat
float64[] thetaHat
int32 estimatorOn
bool usePredictor
bool deadReckoning
bool normalizedKinematics
bool artificialSwitching
bool useVelocityMap
bool streets
bool multiBot
float64 delTon
float64 delToff
